src/components/landing/About.jsx

```diff
@@ -1,5 +1,19 @@
 const About = () => {
-  return <div>About</div>;
+  return (
+    <div className="bg-gradient-to-r from-senior-blue-200 to-senior-blue-100">
```

Comment:
the blue-200 is a little too dark, just add a new blue color that's a little lighter